cocos2dx/platform/CCFileUtils.cpp

```diff
@@ -427,6 +427,60 @@ static tinyxml2::XMLElement* generateElementForDict(cocos2d::CCDictionary *dict,
     return rootNode;
 }
 
+static Data getData(const std::string& filename, bool forString)
```

Comment:
If this new method is only used for OHOS, it's better to add `#if (CC_TARGET_PLATFORM == CC_PLATFORM_OHOS)` restriction.